core/template.js

```diff
@@ -27,28 +24,28 @@
 	 * @param {String} source The template source.
 	 */
 	CKEDITOR.template = function( source ) {
-		// Builds an optimized function body for the output() method, focused on performance.
 		// For example, if we have this "source":
```

Comment:
This comment is a little confusing as the resulting function is not exactly as it describes. The resulting function performs the operations described here, but its body is different.